models/authors.js

```diff
@@ -0,0 +1,9 @@
+const authors = (connection, Sequelize) => {
+  return connection.define('authors', {
+    id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+    nameFirst: { type: Sequelize.STRING, allowNull: false },
+    nameLast: { type: Sequelize.STRING, allowNull: false },
+  }, { defaultScope: { attributes: { exclude: ['deletedAt'] } } }, { paranoid: true })
```

Comment:
Since you are using this defaultScope you don't need attributes in your controllers, it is now redundant.